src/lib/rest/rest.cpp

```diff
@@ -518,6 +518,18 @@ static char* removeTrailingSlash(std::string path)
 */
 int servicePathSplit(ConnectionInfo* ciP)
 {
+  //
+  // Special case: empty service-path 
+  //
+  if ((ciP->httpHeaders.servicePathReceived == true) && (ciP->servicePath == ""))
```

Comment:
If we have fixed something in the code, I think it should mentioned in CHANGES_NEXT_RELEASE.